crates/uv/tests/edit.rs

```diff
@@ -349,13 +517,13 @@ fn add_unnamed() -> Result<()> {
         [[distribution.dependencies]]
         name = "uv-public-pypackage"
         version = "0.1.0"
-        source = "git+https://github.com/astral-test/uv-public-pypackage?rev=0.0.1#0dacfd662c64cb4ceb16e6cf65a157a8b715b979"
+        source = "git+https://github.com/astral-test/uv-public-pypackage?tag=0.0.1#0dacfd662c64cb4ceb16e6cf65a157a8b715b979"
```

Comment:
Odd that the lockfile is different, shouldn't we disambiguate before locking (or merge all to rev)?

Reply:
Opened https://github.com/astral-sh/uv/issues/4417.